### PR TITLE
Update Russian translation

### PR DESCRIPTION
**Pull request by OlesyaGerasimenko**

### CODE REVIEW

**Review by michaelrsweet — CHANGES_REQUESTED** [review]
Huh, still doesn't have the filtering code in it... :/ Are you sure you've updated your fork with the latest changes from OpenPrinting:master?

**Comment by OlesyaGerasimenko** [comment]
Weird, the code seems to be in place: https://github.com/OpenPrinting/cups/pull/852/commits/1f71a7a34973e2bcafbba70bf93e87d150398cf8

**Comment by zdohnal** [comment]
Hi @OlesyaGerasimenko ,

I'm not a Git expert, but I can tell you what works for me when I want to create a pull request:

- clone the upstream (OpenPrinting CUPS) repo on Github,
- in terminal, clone personal OpenPrinting CUPS clone to local machine with f.e. ```git clone git@github.com:zdohnal/cups.git```
- add OpenPrinting CUPS as another remote with ```git remote add upstream https://github.com/OpenPrinting/cups```

This gives me basic setup - I can push into my repo by default and I can always fetch upstream changes from OpenPrinting by ```git pull --rebase upstream master``` and push them to my clone by ```git push```.

Then I create a separate branch by ```git checkout -b <branch-name>``` and push my changes into it - pushing the changes into master may work for one time contribution (if I delete the Github repo once it is merged) if I am lucky - otherwise it can get difficult if there are conflicts and the git history becomes less readable (at least in my case - I always want my local master to be 1-to-1 with upstream master branch, since it shows me what is in upstream more clearly). Once I have committed all changes, I push a new branch by ```git push --set-upstream origin <branch-name>```.

And if there are new changes in upstream master, I have to propagate them into my branch:

- first I take changes from upstream repo - ```git pull --rebase upstream master && git push``` when I am in local master branch of my clone,
- switch to my branch by ```git checkout <branch>```,
- rebase the branch according master branch by ```git rebase -i master```,
- solve any possible conflicts - in case my changes should rewrite upstream changes, remove blocks from upstream (any conflicting line blocks are marked by >>>> and <<<< - those have to be solved) and vice versa. If upstream changes are not related to my changes, I have to adjust the code to contain both (this is what happened to you - upstream changes are not related to your changes, so you had to keep them, but they were removed instead, which is an issue),
- once rebase finishes (there are clear instructions on command line how to proceed during rebase) push the changes with ```git push --force```.

If I check your git history, I see you have tried to merge master branch of OpenPrinting CUPS, but it does not look like recent HEAD of master branch.

If I were you, I would take my changes and delete my cloned github repo, since it would be easiest for me, but in case you would like to keep your repo:

- you can cherry pick your changes by ```git format-patch -1 <commit ID>,
- do hard reset to the commit before you made changes by ```git reset --hard origin <commit ID before you started to make changes> && git push --force```,
- check if you have the remote set by ```git remote -v``` - if not, add it via ```git remote add``` command above,
- update your repo by ```git pull --rebase upstream master && git push```,
- apply your changes and fix any possible conflicts (beware of removing any changes which come from upstream :)),
- Github should now see your changes.

File: templates/ru/printer.tmpl [hunk 1]
```diff
@@ -1,21 +1,21 @@
 <H2 CLASS="title"><A HREF="{printer_uri_supported}">{printer_name}</A>
-({printer_state=3?ожидает:{printer_state=4?печать:приостановлен}},
+({printer_state=3?ожидает:{printer_state=4?выполняет печать:приостановлен}},
 {printer_is_accepting_jobs=0?не принимает задания:принимает задания},
-{server_is_sharing_printers=0?нет совместного доступа:{printer_is_shared=0?нет совместного доступа:разрешен совместный доступ}} {default_name={printer_name}?, По умолчанию:})</H2>
+{server_is_sharing_printers=0?не:{printer_is_shared=0?не:}} использует совместный доступ{default_name={printer_name}?, установлен по умолчанию:})</H2>
```
Comment (ValdikSS) [inline templates/ru/printer.tmpl:4]: Принтер может быть без возможности совместного (общего) доступа (не расшарен по сети для других пользователей), а может быть с совместным доступом (расшарен по сети).

Использует совместный доступ — неверно.

Сам оборот "совместный доступ" не очень звучит, если придумаете, как перефразировать более удачно — дерзайте.

File: templates/ru/printer.tmpl [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
 
 <FORM METHOD="POST" ACTION="{printer_uri_supported}" NAME="maintenance">
 <INPUT TYPE="HIDDEN" NAME="org.cups.sid" VALUE="{$org.cups.sid}">
 <SELECT NAME="OP" ONCHANGE="document.maintenance.submit();">
 <OPTION VALUE="">Обслуживание</OPTION>
 <OPTION VALUE="print-test-page">Печать пробной страницы</OPTION>
 {printer_commands~.*Clean.*?<OPTION VALUE="clean-print-heads">Очистить головки</OPTION>:}
-{printer_commands~.*PrintSelfTestPage.*?<OPTION VALUE="print-self-test-page">Печать пробной страницы принтера</OPTION>:}
-{printer_state=5?<OPTION VALUE="start-printer">Восстановить печать</OPTION>:<OPTION VALUE="stop-printer">Приостановить принтер</OPTION>}
+{printer_commands~.*PrintSelfTestPage.*?<OPTION VALUE="print-self-test-page">Печать страницы самопроверки</OPTION>:}
```
Comment (ValdikSS) [inline templates/ru/printer.tmpl:12]: Doesn't sound right, Windows also uses пробная страница.

File: templates/ru/printer.tmpl [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
+{printer_state=5?<OPTION VALUE="start-printer">Продолжить печать</OPTION>:<OPTION VALUE="stop-printer">Приостановить печать</OPTION>}
```
Comment (ValdikSS) [inline templates/ru/printer.tmpl:13]: "Продолжить" звучит непонятно, хоть и более грамотно.
Предлагаю оставить восстановление — и, возможно, заменить "приостановку" на "удержание", раз вы выше сделали с заданиями такое.

Или просто оставить как есть.

File: templates/ru/printers.tmpl [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {#printer_name=0?:
 <TABLE CLASS="list" SUMMARY="Список принтеров">
 <THEAD>
-<TR><TH>Наименование</TH><TH>Описание</TH><TH>Расположение</TH><TH>Драйвер</TH><TH>Статус</TH></TR>
+<TR><TH>Имя очереди</TH><TH>Описание</TH><TH>Расположение</TH><TH>Марка и модель</TH><TH>Статус</TH></TR>
```
Comment (ValdikSS) [inline templates/ru/printers.tmpl:4]: Производитель, не марка

**Review by ValdikSS — CHANGES_REQUESTED** [review]

File: templates/ru/jobs.tmpl [hunk 5]
```diff
@@ -1,19 +1,19 @@
...
 <TR><TH>Номер</TH><TH>Название</TH><TH>Пользователь</TH><TH>Размер</TH><TH>Страниц</TH><TH>Статус</TH><TH>Управление</TH></TR>
 </THEAD>
 <TBODY>
 {[job_id]
 <TR VALIGN="TOP">
 <TD><A HREF="{job_printer_uri}">{job_printer_name}</A>-{job_id}{?phone? ({phone}):}&nbsp;</TD>
-<TD>{?job_name=?Неизвестное:{job_name}}&nbsp;</TD>
-<TD>{?job_originating_user_name=?Приостановлено пользователем:{job_originating_user_name}}&nbsp;</TD>
+<TD>{?job_name=?Неизвестно:{job_name}}&nbsp;</TD>
+<TD>{?job_originating_user_name=?Приостановка:{job_originating_user_name}}&nbsp;</TD>
```
[the remark this answers is not in the document]
Comment (ValdikSS) [reply templates/ru/jobs.tmpl:11]: @michaelrsweet could you clarify why `Withheld` is used in English here, in `job_originating_user_name`? It always print `Withheld` for me when I use driverless printing.

[the remark this answers is not in the document]
Comment (michaelrsweet) [reply templates/ru/jobs.tmpl:11]: "Withheld" is shown when you don't have authentication enabled, so for privacy reasons we don't provide personally identifying information...

[the remark this answers is not in the document]
Comment (ValdikSS) [reply templates/ru/jobs.tmpl:11]: @michaelrsweet also in .po file, should we translate WARN and FAIL, are these loglevels or just the text?

[the remark this answers is not in the document]
Comment (ValdikSS) [reply templates/ru/jobs.tmpl:11]: > "Withheld" is shown when you don't have authentication enabled, so for privacy reasons we don't provide personally identifying information...

Oh, so it's `unknown` then, gotcha.

[the remark this answers is not in the document]
Comment (michaelrsweet) [reply templates/ru/jobs.tmpl:11]: > @michaelrsweet also in .po file, should we translate WARN and FAIL, are these loglevels or just the text?

The WARN, ERROR, PASS, and FAIL prefixes should not be localized.

[the remark this answers is not in the document]
Comment (ValdikSS) [reply templates/ru/jobs.tmpl:11]: > The WARN, ERROR, PASS, and FAIL prefixes should not be localized.

Just FIY they are localized in ca, es, ja, pt_BR, zn_CN.

**Comment by michaelrsweet** [comment]
Closing this for now, please open a fresh PR once you have resolved the issues.

**Comment by ValdikSS** [comment]
@michaelrsweet, #895 and #894 are updated versions of this translation.

**Pull request closed without merging**